proto/structured.proto

```diff
@@ -27,61 +27,51 @@ option (gogoproto.sizer_all) = true;
 option (gogoproto.marshaler_all) = true;
 option (gogoproto.unmarshaler_all) = true;
 
-message Table {
-  optional string name = 1 [(gogoproto.nullable) = false];
+// For Cockroach internal use.
+message Internal {
+  optional uint32 id = 1 [(gogoproto.nullable) = false];
 }
-
+ 
 message Column {
   enum ColumnType {
     BYTES = 0;
   }
-
   optional string name = 1 [(gogoproto.nullable) = false];
   optional ColumnType type = 2 [(gogoproto.nullable) = false];
+  // For Cockroach internal use.
+  optional Internal internal = 9 [(gogoproto.nullable) = true];
 }
 
 message Index {
   optional string name = 1 [(gogoproto.nullable) = false];
   optional bool unique = 2 [(gogoproto.nullable) = false];
+  // An ordered list of column names of which the index is comprised. Each
+  // column_name refers to a column in the TableSchema's columns. Special
+  // care is taken to update this when deleting columns.
+  repeated string column_names = 3 [(gogoproto.nullable) = false];
+  // For Cockroach internal use.
+  optional Internal internal = 9 [(gogoproto.nullable) = true];
```

Comment:
On the server it will look like internal().id() . Is that really a problem?

I did this to allow us to add more internal stuff later. Would hate to add internal_foo, internal_bar, etc to the protobuf